frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -100,11 +108,37 @@ export class ClusterOverview extends React.Component {
         },
         loaded: false,
       },
+      capacityStats: {
+        stats: {
+          cpu: {
+            title: 'CPU',
+            unit: 'cores',
+            data: {},
+          },
+          memory: {
+            title: 'Memory',
+            unit: 'GB',
+            data: {},
+          },
+          storage: {
+            title: 'Storage',
+            unit: 'Ti',
+            data: {},
+          },
+          network: {
+            title: 'Network',
+            unit: 'GBps',
+            data: {},
+          },
+        },
+        loaded: false, // true if at least one stat is already loaded
```

Comment:
there will be no need to have general loaded state as each 'donut part' could show loading individually

Reply:
Agree - there is no need to explicitly handle state where _no_ stat is loaded.
Rendering just 4 loading components is enough

Reply:
done

Reply:
removed